src/lib/viewers/image/__tests__/ImageViewer-test.js

```diff
@@ -52,11 +53,31 @@ describe('lib/viewers/image/ImageViewer', () => {
         stubs = {};
     });
 
+    describe('destroy()', () => {
+        test('should remove the zoom event listener', () => {
+            image.options.enableAnnotationsImageDiscoverability = true;
+            image.removeListener = jest.fn();
```

Comment:
This seems like it could be a spy rather than an outright stub to preserve the implementation